sharing_portal/templates/sharing_portal/embed.html

```diff
@@ -22,10 +22,6 @@ <h2 class="artifactTitle">{{ form_title }}</h2>
           </div>
         </header>
 
-        {% if version_form %}
```

Comment:
This whole template can go away, we won't need an embedded edit/create form anymore.